File: ios/Plugin/SmsRetrieverPlugin.m
#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

// Define the plugin using the CAP_PLUGIN Macro, and
// each method the plugin supports using the CAP_PLUGIN_METHOD macro.
CAP_PLUGIN(SmsRetrieverPlugin, "SmsRetriever",
           CAP_PLUGIN_METHOD(getAppSignature, CAPPluginReturnNone);
           CAP_PLUGIN_METHOD(requestPhoneNumber, CAPPluginReturnNone);
           CAP_PLUGIN_METHOD(startSmsReceiver, CAPPluginReturnNone);
           CAP_PLUGIN_METHOD(removeSmsReceiver, CAPPluginReturnNone);

)

File: ios/Plugin/SmsRetrieverPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(SmsRetrieverPlugin)
public class SmsRetrieverPlugin: CAPPlugin {
    
    @objc func getAppSignature(_ call: CAPPluginCall) {
        call.unimplemented()
    }
    
    @objc func requestPhoneNumber(_ call: CAPPluginCall) {
        call.unimplemented()
    }
    
    @objc func startSmsReceiver(_ call: CAPPluginCall) {
        call.unimplemented()
    }
    
    @objc func removeSmsReceiver(_ call: CAPPluginCall) {
        call.unimplemented()
    }
}
